src/content/en/2020/seo.md

```diff
@@ -83,620 +83,714 @@ Response Code | All Devices | Mobile | Desktop
 
-### Web Performance
+### Web performance
 
-Having a fast loading website is fundamental to provide a great user search experience. Because of its importance it has been taken into consideration as a ranking factor by search engines since many years ago. Google initially announced using site speed as a [ranking factor since 2010](https://webmasters.googleblog.com/2010/04/using-site-speed-in-web-search-ranking.html), and then [in 2018 did the same for mobile searches](https://webmasters.googleblog.com/2018/01/using-page-speed-in-mobile-search.html).
-
-As of November 2020, 3 performance metrics known as [Core Web Vital](https://webmasters.googleblog.com/2020/05/evaluating-page-experience.html) are roadmapped to be a ranking factor as part of the "page experience" signals since May 2021. Sites should expect Googlebot to crawl using its [mobile user-agent](https://support.google.com/webmasters/answer/1061943) with the [Chromium](https://webmasters.googleblog.com/2019/10/updating-user-agent-of-googlebot.html) version details updating as the web Rendering Service updates with new Chromium releases.
-
-#### Lighthouse v6 and Web Core Vitals
-
-
-For SEOs, performance disambiguated from "speed" with changes to Lighthouse's core measurement methodology in update to version 6.  Lighthouse is the scoring tool that populates key SEO performance tools such as web.dev, Chrome DevTools, and PageSpeed Insights.
-Previously, performance scores and methods of measurement varied by tool.  Google announced a set of unified performance metrics called [Core Web Vitals](https://developers.google.com/search/blog/2020/05/evaluating-page-experience) in May 2020.
-
-Each of the metrics aligns to a phase in a user's experience.  The data source is the [Chrome User Experience Report](https://developers.google.com/web/tools/chrome-user-experience-report/) (Crux).  This field data is aggregated from users who have opted-in to syncing their browsing history, have not set up a Sync passphrase, and have usage statistic reporting enabled.
-
-Core Web Vitals consist of:
+Having a fast-loading website is fundamental to provide a great user search experience. Because of its importance, it has been taken into consideration as a ranking factor by search engines for years. Google initially announced using site speed as a [ranking factor in 2010](https://webmasters.googleblog.com/2010/04/using-site-speed-in-web-search-ranking.html), and then [in 2018 did the same for mobile searches](https://webmasters.googleblog.com/2018/01/using-page-speed-in-mobile-search.html).
 
+{# NOTE(authors): I've made some ruthless edits to this section to remove everything related to synthetic measurement of CWV, including the entire Lighthouse discussion, which is orthogonal to the real-user aspect of CWV. Please push back if you disagree with any of these edits. #}
```

Comment:
⚠️ @aleyda FYI this was a significant edit. I wanted to makes sure the CWV discussion was focused on real-user data from CrUX and not get it confused with lab data from Lighthouse.

Reply:
Thanks for the heads up @rviscomi ! @fellowhuman1101 since these are major editions from the performance section, could you please take a look too in case there's something you would like to change/replace? :) I'll check it out too but want to make sure you're ok with them!

Reply:
@aleyda Confirming-- no changes requested. I think @rviscomi 's edits are judicious and help disambiguate Crux data's role as a ranking factor from Lighthouse's lab data.

Reply:
Thanks @fellowhuman1101, that's great - I just took a look too, it's all good @rviscomi :)